stripe-node/app.js

```diff
@@ -52,31 +50,53 @@ app.post('/charge/angular', (req, res) => {
     source: req.body.token.id,
     shipping: address,
     name: req.body.token.name,
-  })
-  // Add charge to customer
-    .then(customer => stripe.charges.create({
+  }, (err, customers) => {
```

Comment:
Is `customers` an array? Why the plural?

Reply:
good point. I'll leave it as customer.